app/views/school/internet/mobile/bulk_requests/new.html.erb

```diff
@@ -28,6 +28,15 @@
       label: { text: 'Pick a spreadsheet file', size: 'm' },
       accept: Mime::Type.lookup_by_extension(:xlsx)
     %>
+
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        <%= MobileNetwork.excluded_fe_networks.pluck(:brand).to_sentence(last_word_connector: ' and ') %> will not accept requests for students over the age of 16
+      </strong>
+    </div>
+
```

Comment:
This block should only show to schools/colleges etc that we know have students over 16

Reply:
this can't be done at the moment and we will have to show for all. we simply do not have the data for it. there is a backlog ticket titled `import `StatutoryLowAge` from GIAS`. if we do that first we can use that as the toggle for this